shows/430 - Creator of Wordle.md

```diff
@@ -0,0 +1,81 @@
+---
+number: 430
+title: Creator of Wordle Josh Wardle
```

Comment:
Can you make this Creator of Wordle - Josh Wardle

Reply:
Done!